tests/orders/ordersClient.test.ts

```diff
@@ -141,4 +146,25 @@ describe('OrdersClient', () => {
       expect(result.toJSON()).to.eql(PAYLOAD_ORDERS_WITHOUT_OPTIONAL);
     });
   });
+
+  describe('saveOrderEavs', () => {
+    it('calls the saveOrderEavs method', async () => {
+      const orderReference = 'XSP12345';
+      const saveOrderEavsData: SaveOrderEavsInputType = {
+        eavs: [
+          {
+            eavkeyName: 'order_comment_1',
+            tableName: 'ORDER_INFO',
+            value: 'The comment one',
+          },
+        ],
+      };
+
+      nock(ORDERS_MOCK_URL)
+        .post(`/orders/${orderReference}/saveOrderEavs`)
+        .reply(201);
+
+      await client.saveOrderEavs(orderReference, saveOrderEavsData);
```

Comment:
You can check if `saveOrderEavs` call properly the nock service with this : 

`expect(nock.isDone()).to.be.true`